payments-core/src/main/java/com/stripe/android/IntentConfirmationInterceptor.kt

```diff
@@ -57,7 +57,7 @@ interface IntentConfirmationInterceptor {
     companion object {
         var createIntentCallback: AbsCreateIntentCallback? = null
 
-        const val DISMISS_WITH_SUCCESS = "DISMISS_WITH_SUCCESS"
+        const val COMPLETE_WITHOUT_CONFIRMING_INTENT = "COMPLETE_WITHOUT_CONFIRMING_INTENT"
```

Comment:
Should this be marked with `@DelicatePaymentSheetApi`? That's what I'm seeing in the API review.